### PR TITLE
Initialize component

### PR DESCRIPTION
**Pull request by DuaneDave**
### In this PR, we added:

- A book component
- A form component
- A booklist component to render all books
- A global button component configurable from outside the component
- A global input component configurable from outside the component

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
+import './index.css';
 
 import Header from './components/header/Header';
 import BookList from './components/books/BookList';
 import Form from './components/form/Form';
 
 function App() {
   return (
-    <Fragment>
+    <>
```
Comment (rustam-tolipov) [inline src/App.js:11]: You made a great effort for making this project. However, there are some implementation need to be added.

- [ ] So alongside your Books component, you need to have a Categories component. Kindly add the Categories comp.. to your project. 

In categories: 

> Categories
> 
> should display only a button with the text "Check status"

File: src/App.js [hunk 2]
```diff
@@ -1,16 +1,17 @@
...
       <Header />
       <BookList />
       <Form />
-    </Fragment>
```
Comment (rustam-tolipov) [inline src/App.js:13]: - [ ] Also, you should use `Route` and `NavLink` to navigate through your Books and Categories.

File: src/UI/Input.jsx [hunk 3]
```diff
@@ -1,4 +1,3 @@
-/* eslint-disable */
 function Input({ type, placeholder }) {
   return (
```
Comment (rustam-tolipov) [inline src/UI/Input.jsx:3]: [OPTIONAL] Kindly add MIT license to your project.

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hello @DuaneDave, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (DuaneDave) [reply src/App.js:11]: Hello @Rustamxon7 

I'm sorry but this is not a requirement for this project, I can not find where it was clearly stated to have a component called category. 

Please review the requirements [see here](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/project_init_project_wth_comps_v2_0.md#:~:text=Should%20have%20a,with%20an%20author.%5D)

Comment (DuaneDave) [reply src/App.js:11]: Don't mind me, I have seen it.

I'm sorry for the confusion

Comment (peterrobert) [reply src/App.js:11]: This change has been done well

File: src/App.js (hunk 2 above)
Comment (peterrobert) [reply src/App.js:13]: This change needs some more work Kindly check the review

File: src/UI/Input.jsx (hunk 3 above)
Comment (peterrobert) [reply src/UI/Input.jsx:3]: This change has been done well.

**Review by peterrobert — CHANGES_REQUESTED** [review]
Hi @DuaneDave, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

 ## Required Changes ♻️

- [ ]  When I try to run your project locally, this error is appearing `Can't resolve 'react-router-dom`. It seems you didn't add react-router-dom as a dependency to your project. To learn more about this kindly follow this [article](https://www.freecodecamp.org/news/react-router-in-5-minutes/)

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏


Feel free to leave any questions or comments in the PR thread or tag me @peterrobert if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @DuaneDave , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**